kf/bpf.go

```diff
@@ -534,56 +506,64 @@ func (b *BPF) UpdateArgs(ifaceName, direction string) error {
 }
 
 // Status of user program is running
-func (b *BPF) isRunning() (bool, error) {
-	// No user program or may be disabled
+func (b *BPF) isRunning() (bool, bool, error) {
+	userProgram := true
+	bpfProgram := false
+	var err error
+
+	// CmdStatus should check for user and BPF program
 	if len(b.Program.CmdStatus) > 1 {
 		cmd := filepath.Join(b.FilePath, b.Program.CmdStatus)
 
 		if err := assertExecutable(cmd); err != nil {
-			return false, fmt.Errorf("failed to execute %s with error: %v", b.Program.CmdStatus, err)
-		}
-
-		args := make([]string, 0, len(b.Program.StatusArgs)<<1)
+			userProgram = false
+		} else {
+			args := make([]string, 0, len(b.Program.StatusArgs)<<1)
 
-		for k, val := range b.Program.StatusArgs {
-			if v, ok := val.(string); !ok {
-				err := fmt.Errorf("status args is not a string for the ebpf program %s", b.Program.Name)
-				log.Error().Err(err).Msgf("failed to convert status args value into string for program %s", b.Program.Name)
-				return false, err
+			for k, val := range b.Program.StatusArgs {
+				if v, ok := val.(string); !ok {
+					err = fmt.Errorf("status args is not a string for the ebpf program %s", b.Program.Name)
```

Comment:
```suggestion
					err = fmt.Errorf("status args is not a string for the bpf program %s", b.Program.Name)
```
nit: other places in this file it's called the "bpf" program... be consistent.